README.md

```diff
@@ -17,10 +17,10 @@ following clients and servers:
 
 - Connect, using [Connect's Go implementation][connect-go]
```

Comment:
now that we have `connect-web` should we just have this be `connect-go`? and then change the following bullet to `grpc-go` since we also have the grpc-web client?

Reply:
yeh updated to make it more clear and aligned